### ISSUE TITLE
Should add the Foot Shooting Prevention Agreement as a README

### ISSUE THREAD

**Issue by harman28** [moserware/AES-Illustrated#2]
A copy of the Foot Shooting Prevention Agreement should be present in the repo as a README, complete with the interpretive dance clause. Could open a PR for this.

**Comment by WiseWomanVP** [comment]
Yes, indeed! I have to add that in my learning management system when I have my students study this code :)
